console/services/market_app_service.py

```diff
@@ -1152,7 +1152,7 @@ def get_rainbond_app_and_versions(self, enterprise_id, app_id, page, page_size):
 
         app = app.to_dict()
         app["tags"] = tag_list
-        app_versions.sort(lambda x, y: cmp(x["version"], y["version"]))
+        app_versions.sort(lambda x, y: cmp(y["version"], x["version"]))
```

Comment:
Can you compare sizes directly by version number string?

Reply:
First of all, the format is limited when the version is released, for example, 1.0 / 1.0.1. Second, only one record will be retrieved for the same version according to the latest update time, so the same version will not be repeated. Therefore, I think it is feasible to sort by version number.